examples/fixture/asm/kimchi/public_output_bool.asm

```diff
@@ -0,0 +1,29 @@
+@ noname.0.7.0
+
+DoubleGeneric<1>
+DoubleGeneric<1>
```

Comment:
there's only 2 public inputs here but we should have expected 4 right? (doesn't help that we don't have https://github.com/zksecurity/noname/issues/65)